user/snapper/patches/ncurses.patch

```diff
@@ -0,0 +1,10 @@
+diff --git a/client/utils/Makefile.am b/client/utils/Makefile.am
+index e6a9968..aeeea5a 100644
+--- a/client/utils/Makefile.am
++++ b/client/utils/Makefile.am
+@@ -20,4 +20,4 @@ libutils_la_SOURCES =					\
+ 	CsvFormatter.cc	    	CsvFormatter.h		\
+ 	JsonFormatter.cc    	JsonFormatter.h
+ 
+-libutils_la_LIBADD = ../../snapper/libsnapper.la -ltinfo
++libutils_la_LIBADD = ../../snapper/libsnapper.la -lncurses
```

Comment:
this isn't needed, libtinfo compat symlinks exist, they're just in ncurses-libtinfo-devel for some reason, i'll put them in the main one